### ISSUE TITLE
Dateselected event is triggered twice when selecting few calendar dates in a calendar control

### ISSUE THREAD

**Issue by pradeeparutla** [rebeccaXam/XamForms.Controls.Calendar#31]
When I am using 5 different calendar controls to display more than once calendar month, I have implemented your suggestion using the calendar view model to select only one date at a time in all the calendar controls. I am facing one new issue in android devices. Sometimes Date Clicked event is hitting twice on selecting few dates in a given month. For February 2016, when selecting the dates from1st Feb 2016 to 6th Feb, DateClicked event is triggered twice, where as from Feb 7th to 27th 2016 is triggered only once. 
Note: I am seeing this issue after implementing Calendar view model for selecting only one date at a time in all the calendar controls. This is in reference to #25

**Comment by pradeeparutla** [comment]
Hi Rebacca. Please let me know if there is a solution for this issue. Waiting for long time on this.

**Comment by rebeccaXam** [comment]
Since version 1.0.8 you can use one calendar control and show multiple month with 
``ShowNumOfMonths = 5`` and it will show 5 months in a Scollview:

<img width="378" alt="screen shot 2017-04-26 at 13 13 04" src="https://cloud.githubusercontent.com/assets/19565685/25431752/2b108072-2a82-11e7-89ce-fed41d0df675.png">

It will automatically only select one date when ``MultiSelectDates = false;``.

**Comment by pradeeparutla** [comment]
Okay. Thankyou for the suggestion.
